gradle.properties

```diff
@@ -16,7 +16,7 @@ org.gradle.vfs.verbose=true
 org.gradle.vfs.debug=true
 
 # If you're experimenting with changes and don't want to update the verification file right away, please change the mode to "lenient" (not "off")
-org.gradle.dependency.verification=strict
+org.gradle.dependency.verification=lenient
```

Comment:
:x: Don't forget to remove this before merging

Reply:
Yes. I still have to update and check the verification metadata (TODO is in the description). Just wanted to make sure we want to go with this before doing it.